string idTask
string idSeller
string idBuyer
float64 price
---
bool status
